### DISCUSSION TITLE
Improved Town Industries의 일부 화물 호환 여부

### DISCUSSION THREAD

**Discussion by deholic** [KoreanGRF/KoreanTrainSet#482] [질문(Q&A)] [answered]
[Improved Town Industries](https://github.com/2TallTyler/improved_town_industries)의 화물 중 하나인 Scrap Paper를 실을 수 있는 화물차가 없는 것으로 보입니다.
이건 어느 쪽에다 문의를 드려야 하는 부분일까요? 

![스크린샷 2024-11-27 06 36 21](https://github.com/user-attachments/assets/3d14d9af-2eaa-4ec4-ae55-22fb9949b4f4)

**Comment by telk5093** [comment]
여기에 물어보시는 게 맞습니다.
Improved Town Industries는 아직 지원하고 있지 않아서, 지원을 위해서는 좀 살펴볼 필요가 있습니다.

**Comment by telk5093** [comment, marked as answer]
[d3752f7](https://github.com/KoreanGRF/KoreanTrainSet/commit/d3752f7963de34a67ccb8171d8def9986a96a40f) 에서 반영되었으며 다음 버전부터 사용 가능할 예정입니다.
